src/lib/fetch.d.ts

```diff
@@ -0,0 +1,129 @@
+/// The fetch API standard can be found at https://fetch.spec.whatwg.org/
+/// Definitions by: Ryan Graham <https://github.com/ryan-codingintrigue>, Kagami Sascha Rosylight <https://github.com/saschanaz>, Robin Van den Broeck <https://github.com/gamesmaxed>

```

Comment:
We have not included any similar headers in the other lib files. please remove that.